.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: cpp
+
+# enabling multi platform build (under travis beta)
+# requires emailing support@travis-ci.com for this to be enabled for our repo
+# http://docs.travis-ci.com/user/multi-os/
+matrix:
+    include:
+        - os: osx
+          compiler: clang
+        - os: linux
+          compiler: gcc
+
+before_install:
+    - git submodule init && git submodule update
+    - source ./travis/before_install.sh
+
+install: ulimit -c
+
+before_script: 
+    # Set the core file limit to unlimited so a core file is generated upon crash
+    - ulimit -c unlimited -S
+
+# build test project
+script: 
+    - source ./travis/script_build_tests.sh
+    - source ./travis/script_run_tests.sh
+    - source ./travis/script_build.sh
+
+branches:
```

Comment:
Once merged with master, will remove this directly in master!.

If I do now all pushes on all branches will trigger builds and fail.